File: perl/anagram.pl
#!/bin/perl


print("enter first string:");

chomp(my $name=<STDIN>);
my @arr = split //,$name;
print("enter second string:");
chomp(my $name2=<STDIN>);
my @arr2 = split //,$name2;

my $store=join('', sort @arr);
my $store2=join('', sort @arr2);
	if ($store eq $store2) {
	   print("this is anagram","\n");
	}
	else{
	  print("this is not anagram","\n");
	}

File: python/anagram.py

	#!/usr/bin/env python3

	name = input("enter first string: ")
	arr = sorted(name)
	store = ''.join(arr)

	name2 = input("enter second string: ")
	arr2 = sorted(name2)
	store2 = ''.join(arr2)

	if store == store2:
		print("this is anagram")
	else:
		print("this is not anagram")

